apps/web/components/eventtype/AvailabilityTab.tsx

```diff
@@ -81,6 +79,16 @@ const AvailabilitySelect = ({
       : option.value === schedules.find((schedule) => schedule.isDefault)?.id
   );
 
+  useEffect(() => {
+    if (value) {
+      formMethods.setValue("availability", value);
+    }
+  }, [value?.value]);
```

Comment:
Let's avoid this pattern. Let's either pass an async defaultValue to the form or don't render this until the value is defined.